src/System.Management.Automation/engine/Modules/ModuleCmdletBase.cs

```diff
@@ -2363,18 +2363,60 @@ private IEnumerable<PSModuleInfo> CreateFakeModuleObject(IEnumerable<ModuleSpeci
             bool isConsideredCompatible = ModuleUtils.IsPSEditionCompatible(moduleManifestPath, inferredCompatiblePSEditions);
             if (!BaseSkipEditionCheck && !isConsideredCompatible)
             {
-                if (importingModule)
+                if (PowerShellConfig.Instance.IsImplicitWinCompatEnabled())
                 {
-                    IList<PSModuleInfo> moduleProxies = ImportModulesUsingWinCompat(new string [] {moduleManifestPath}, null, new ImportModuleOptions());
+                    if (importingModule)
+                    {
+                        IList<PSModuleInfo> moduleProxies = ImportModulesUsingWinCompat(new string [] {moduleManifestPath}, null, new ImportModuleOptions());
 
-                    // we are loading by a single ManifestPath so expect max of 1
-                    if(moduleProxies.Count > 0)
+                        // we are loading by a single ManifestPath so expect max of 1
+                        if(moduleProxies.Count > 0)
+                        {
+                            return moduleProxies[0];
+                        }
+                        else
+                        {
+                            return null;
+                        }
+                    }
+                }
+                else
```

Comment:
Is the code in `else` block here previously existed and got removed in the WinCompatModule PR, and now brought back?

Reply:
Yes; this `else` branch was there when experimentalFeature check failed. This `else` branch was removed by `Make approved features non-experimental` PR.